packages/commonwealth/client/styles/components/react_quill/markdown_formatted_text.scss

```diff
@@ -12,78 +13,198 @@
   @include hidden-formatting();
 
   width: 100%;
+  overflow: visible;
 
   p {
+    @include b2;
     display: block;
     margin-block-start: 1em;
     margin-block-end: 1em;
     margin-inline-start: 0;
     margin-inline-end: 0;
   }
 
-  overflow: visible;
+  a {
+    @include b2;
+    color: $primary-500;
+  }
+
+  h1 {
+    font-family: $font-family-silka;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 36px;
+    letter-spacing: -0.24px;
```

Comment:
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/1c7af9bc-ef7b-4a06-a876-361a0b89a608)

Sizing of headers does not seem to be accurate. Per ticket `Headers 1-5 should reflect our design system`. 

Figma with headers: https://www.figma.com/file/yCUKk46hg9Fy3EXGkSoaKT/%F0%9F%9A%A7-Foundation?type=design&node-id=31-178&mode=design&t=htdDyuWwikW9yR7X-0

Reply:
cc @Miaplacidus then this PR is good to go

Reply:
@jnaviask @masvelio 
The description is a bit confusing. Per a conversation between myself, @zakhap, and @jessmart1213, the header sizing is per the Figma doc attached to the ticket. In addition, headers 3 - 6 all have the same styling.